doc/sphinxman/source/build_planning.rst

```diff
@@ -1250,7 +1250,7 @@ commonly set up through the :envvar:`PSI_SCRATCH` environment variable:
 
 .. code-block:: bash
 
-    # sh, bash: add to shell or ~/.bashrc (Linux) or ~/.bash_profile (Mac) file
+    # sh, bash: add to shell or ~/.bashrc (Linux/Windows) or ~/.bash_profile (Mac) file
```

Comment:
Indeed. But David tried it the night before FAME and it just worked (increasing font size for projection seems to be the only missing piece), so I figured it was time to acknowledge.